clients/githubrepo/tarball.go

```diff
@@ -70,23 +72,18 @@ type tarballHandler struct {
 func (handler *tarballHandler) init(ctx context.Context, repo *github.Repository) error {
 	// Cleanup any previous state.
 	if err := handler.cleanup(); err != nil {
-		return fmt.Errorf("error during githubrepo cleanup: %w", err)
+		return err
```

Comment:
Why remove the `fmt.Errorf` here? Will be useful to have some context for the errors.

Reply:
ah, I see what you mean. My brain inaccurately flagged this because of `Errorf` + `%w`. I agree this usage is fine. Do you think I should add a dedicated function in `errors.go` for this case? I don't know. if that's even useful. Le me know what you think.

Reply:
We can skip the dedicated errors for this case. These anyways get exposed here: https://github.com/ossf/scorecard/blob/main/pkg/scorecard.go#L87

Reply:
gocha!